src/app/profile/spaces/space.service.ts

```diff
@@ -108,6 +108,8 @@ export class SpaceService {
   private buildSpaceIndexMap(): void {
     this.spaces.forEach((space, index) => {
       this.spaceIdIndexMap[space.id] = index;
+      // TODO Hack in the path
+      space.path = space.attributes.name;
```

Comment:
The attributes of the Space model class will eventually have to match the one of the Spaces resource in the backend. It looks like we'll have to move `path` to a computed property of a new UI model class, or compute path from `name`. Given that we need only the space name to query for one from the backend and display it on changing a route, I'm not sure we need to retain `path`.